logic/tours/scrapping_single_tour.py

```diff
@@ -44,6 +44,20 @@ def get_tour_name(driver: WebDriver) -> Optional[str]:
         return None
 
 
+def get_tour_rating(driver: WebDriver) -> Optional[str]:
+    rating_xpath = ("/html/body/div[5]/div[4]/div[2]/div/div[1]/div[3]/div[1]/div[2]/div[2]/span/span[2]/span[2]"
+                    "/span[2]/div[2]/div/span[2]/span[2]/div/div[1]/span/strong")
+    rating = scrape_element_by_xpath(driver, rating_xpath)
+    return rating
+
+
+def get_tour_description(driver: WebDriver) -> Optional[str]:
+    description_xpath = ("/html/body/div[5]/div[4]/div[2]/div/div[1]/div[3]/div[1]/div[5]/div/section/span/span[2]"
+                         "/div/div/p/text()")
```

Comment:
`text()` seems suspicious for an Xpath, I think extracting text should / *might* be done using `WebElement.text`